### PR TITLE
Reorganize docs

### PR DESCRIPTION
**Pull request by rikvl**

### CODE REVIEW

File: docs/tutorials/single_screen.rst [hunk 1]
```diff
@@ -37,25 +43,20 @@ Start with some standard imports.
...
+The `hsv` colormap from :py:mod:`matplotlib.cm` is used as fallback.
 
-.. plot::
-    :include-source:
-    :context:
+.. jupyter-execute::
 
     try:
-        from hue_cycle_cmap import cmap as phasecmap
+        from screens.visualization import phasecmap
```
Comment (mhvk) [inline docs/tutorials/single_screen.rst:52]: Can this import still fail?

**Review by mhvk — COMMENTED** [review]
This looks good. Just one question in-line.

File: docs/tutorials/single_screen.rst (hunk 1 above)
Comment (rikvl) [reply docs/tutorials/single_screen.rst:52]: This way, one can run the downloadable script/notebook without having `screens` installed.

Comment (mhvk) [reply docs/tutorials/single_screen.rst:52]: OK, that's fine (though perhaps one can insist on having screens in the screens documentation!)

**Pull request merged**